manager/deks.go

```diff
@@ -156,8 +165,9 @@ func (r *RaftDEKManager) NeedsRotation() bool {
 }
 
 // GetKeys returns the current set of DEKs.  If NeedsRotation is true, and there
-// is no existing PendingDEK, it will try to create one.  If there are any errors
-// doing so, just return the original.
+// is no existing PendingDEK, it will try to create one.  If it successfully creates
```

Comment:
The name 'GetKeys' implies a read-only operation so I would suggest avoid doing operations within this function that have side effects.

Reply:
I'd be happy to revisit this at a later point, but that's probably beyond the scope of this PR.